src/main/java/core/basesyntax/bookstore/BookstoreApplication.java

```diff
@@ -20,29 +20,28 @@ public static void main(String[] args) {
 
     @Bean
     public CommandLineRunner commandLineRunner() {
-        return new CommandLineRunner() {
-            @Override
-            public void run(String... args) throws Exception {
-                Book shining = new Book(
-                        "The Shining",
-                        "Stephen King",
-                        "9781444720723",
-                        BigDecimal.valueOf(645),
-                        "spooky book",
-                        "https://m.media-amazon.com/images/I/81zqohMOk-L.jpg"
-                );
-                Book bot = new Book(
-                        "Bot",
-                        "Max Kidruk",
-                        "9789661446532",
-                        BigDecimal.valueOf(300),
-                        "techno thriller",
-                        "https://i.ebayimg.com/images/g/jRwAAOSwP~tW5EgU/s-l1200.jpg"
-                );
-                System.out.println(bookService.save(shining));
-                System.out.println(bookService.save(bot));
-                System.out.println(bookService.findAll());
-            }
+        return args -> {
+            CreateBookRequestDto shining = new CreateBookRequestDto(
+                    "The Shining",
+                    "Stephen King",
+                    "9781444720723",
+                    BigDecimal.valueOf(645),
+                    "spooky book",
```

Comment:
you can remove it for now and use endpoints

Reply:
I keep it to have non empty DB on start-up (also going to do same thing with other entities for development period)